### PR TITLE
[VE] Add unittest for intrinsics

### PR DESCRIPTION
**Pull request by kaz7**
Add unittest for VE intrinsics instructions.

### CODE REVIEW

File: llvm/unittests/Target/VE/MachineInstrTest.cpp [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+using namespace llvm;
+
+TEST(VETest, VLIndex) {
+  using namespace VE;
+
+  // Return expected VL register index in each MI's operands.  Aurora VE has
+  // multiple instruction formats for each instruction.  So, we defines
+  // instructions hierarchically and tests a part of whole instructions.
+  // This function returns -1 to N as expected index, or -2 as default.
+  // We skip a test on an instructions that this functions returns -2.
```
Comment (efocht) [inline llvm/unittests/Target/VE/MachineInstrTest.cpp:28]: "We skip a test on an instruction if this function returns -2."

**Review by efocht — CHANGES_REQUESTED** [review]
Nothing to complain except the slightly unclear comments.

File: llvm/unittests/Target/VE/MachineInstrTest.cpp [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+#include "gtest/gtest.h"
+
+using namespace llvm;
+
+TEST(VETest, VLIndex) {
+  using namespace VE;
+
+  // Return expected VL register index in each MI's operands.  Aurora VE has
+  // multiple instruction formats for each instruction.  So, we defines
+  // instructions hierarchically and tests a part of whole instructions.
```
[the remark this answers is not in the document]
Comment (kaz7) [reply llvm/unittests/Target/VE/MachineInstrTest.cpp]: Thanks!  Corrected.

File: llvm/unittests/Target/VE/MachineInstrTest.cpp (hunk 1 above)
Comment (kaz7) [reply llvm/unittests/Target/VE/MachineInstrTest.cpp]: Updated.

**Review by efocht — APPROVED** [review]

**Pull request merged**